regression/cbmc/bad_option/main.c

```diff
@@ -0,0 +1,11 @@
+void exit(int status);
```

Comment:
You're right, I just took a random file from the test suite. I'll make simpler for it be less misleading.